presto-docs/src/main/sphinx/connector/singlestore.rst

```diff
@@ -68,9 +68,12 @@ that catalog name instead of ``singlestore`` in the above examples.
 SingleStore Connector Limitations
 ---------------------------------
 
+For :doc:`/sql/create-table` statement, the default table type is columnstore.
+The table type can be configured by setting the ``default_table_type`` engine variable, see the
+`documentation <https://docs.singlestore.com/cloud/create-a-database/creating-a-columnstore-table/>`_.
```

Comment:
```suggestion
`Creating a Columnstore Table <https://docs.singlestore.com/cloud/create-a-database/creating-a-columnstore-table/>`_.
```
A general rule for links is to use informative content in the destination label, and one good way to do this is to use the title of the page the user will see when they open the link.